### ISSUE TITLE
Change entity without calling init again

### ISSUE THREAD

**Issue by hambj** [actito/data-model-sdk#7]
At the moment, the entity is chosen when calling "init".

If the webservices user has access to more than one entity, init must be called again to select the new entity.

Suggestion 1 : 
* remove the entity from init
* choose the entity in the callers of the http.ts methods
  * e.g. createProfile must put "/entity/someentity" in the path before "/table/..." when calling actitoPost

Suggestion 2 :
* keep the entity in init
* change the actitoFetch method to return a function with an optional _entity_ argument
* add a second actitoFetch method which calls the previous one with the entity from actitoCredentials (this would be the default behaviour)

@alaindresse , your opinion on this ?

FYI, a solution to this issue would be a nice-to-have for the "automate" project. With a somewhat extremist point of view, we could extend 'Suggestion 2' to the user and password as well.

**Comment by alaindresse** [comment]
I would go for suggestion 2. For the vast majority of cases, we will use a single entity, so I like the idea that a default can be set. Extending on your Idea, I would suggest to allow any variable from the credentials to be overridded with an optional parameter in the calls.
Something like :

``` typescript
functions xyz (overrideCredentials ?: IActitoCredentials = {}){
   const credentials = {...defaultCredentials, ...overrideCredentials);
   ...
}
```

**Comment by hambj** [comment]
Ok for change, done and merged in #10
